test/dbus/lvmdbustest.py

```diff
@@ -1702,6 +1726,29 @@ def test_tag_regression(self):
 			tag in vg_proxy.Vg.Tags,
 			"%s not in %s" % (tag, str(vg_proxy.Vg.Tags)))
 
+	def _verify_existence(self, cmd, operation, resource_name):
+		ec, stdout, stderr = call_lvm(cmd)
+		if ec == 0:
+			path = self._lookup(resource_name)
+			self.assertTrue(path != '/')
+		else:
+			self.assertTrue(ec == 0, "%s exit code = %d" % (operation, ec))
+			std_err_print(
```

Comment:
Does this ``std_err_print()`` ever have a chance to execute? AFAIK, ``self.assertTrue`` raises an exception.